src/index.html

```diff
@@ -1,13 +1,297 @@
+                      alt="Bike front photo"
+                    >
+                  </a>
+
+                  <h3 class="detail__title">Range & Integrated lights</h3>
+                  <p class="detail__description">
+                    The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+                  </p>
+                </div>
+              </article>
+
+              <article class="detail">
+                <div class="detail__section">
+                  <a href="#" class="detail__link detail__link--square">
+                    <img
+                      class="detail__photo"
+                      src="images/details/brakes.png"
+                      alt="Bike brakes photo"
+                    >
+                  </a>
+
+                  <a href="#" class="detail__link detail__link--wide">
+                    <img
+                      class="detail__photo"
+                      src="images/details/foot-board.png"
+                      alt="Bike footboard photo"
+                    >
+                  </a>
+                  <h3 class="detail__title">Hydraulic disc brakes & Lightweight</h3>
+                  <p class="detail__description">
+                    Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+                  </p>
+                </div>
+              </article>
+
+              <div class="detail__section">
+                <button class="detail__button">Explore</button>
+              </div>
+            </div>
+          </section>
+
+          <section class="contacts" id="contacts">
+            <h2 class="section-title">Contact us</h2>
+            <div class="contacts__wrapper">
+              <form
+                class="contacts__form"
+              >
+                <label class="contacts__label-name">
+                  <input
+                    class="contacts__name"
+                    type="text"
+                    placeholder="Name"
+                    required
+                  >
+                </label>
+
+                <label class="contacts__label-email">
+                  <input
+                    class="contacts__email"
+                    type="email"
+                    placeholder="Email"
+                    required
+                  >
+                </label>
+
+                <label class="contacts__label-message">
+                  <textarea
+                    class="contacts__message"
+                    name="message"
+                    rows="8"
+                    placeholder="Message"
+                    required
+                  ></textarea>
+                </label>
+
+                <button class="contacts__button">Send</button>
```

Comment:
```suggestion
                <button type="submit" class="contacts__button">Send</button>
```
make it the `submit` button to make form.onsubmit function works when you click this button